code/obj/item/clothing/armor.dm

```diff
@@ -510,4 +510,8 @@ TYPEINFO(/obj/item/clothing/suit/armor/NT_alt)
 		..()
 		setProperty("meleeprot", 5)
 		setProperty("rangedprot", 0.5)
+	attackby(obj/item/W, mob/user)
```

Comment:
Any reason for this not to be a direct descendant of `/obj/item/clothing/suit/armor` instead? This still has the property of being usable for hazard suit armoring, but it's unclear if that is intentional (and if so, it appears bugged from quick testing).

Reply:
yea fair. i hotfixed it super quick but this is definitely smarter